Makefile

```diff
@@ -222,32 +236,24 @@ examples:
 	$(MAKE) -C internal/examples
 
 .PHONY: crossdock
-crossdock: $(_BIN_DIR)/docker-compose
-	PATH=$(_BIN_DIR):$$PATH docker-compose kill go
-	PATH=$(_BIN_DIR):$$PATH docker-compose rm -f go
-	PATH=$(_BIN_DIR):$$PATH docker-compose build go
-	PATH=$(_BIN_DIR):$$PATH docker-compose run crossdock
+crossdock: $(DOCKER_COMPOSE)
+	PATH=$(BIN):$$PATH docker-compose kill go
+	PATH=$(BIN):$$PATH docker-compose rm -f go
+	PATH=$(BIN):$$PATH docker-compose build go
+	PATH=$(BIN):$$PATH docker-compose run crossdock
```

Comment:
If we have the full path to the executable, do we really need to prepend the directory to the PATH? Can't we do,

    $(DOCKER_COMPOSE) kill go